### ISSUE TITLE
WCAG2Mobile: Draft feedback - 1.3.1 Info and Relationships

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/wcag2ict#646]
(Mary Jo) 

- If WCAG2ICT makes the suggested change, we can check to make sure this language would resolve the MATF issue #1.

**Comment by pday1** [comment]
Closed - insufficient time or resource to review each WCAG2Mobile clause
